Update spesifikasjon-felter.adoc

jimjyang opened the pull request:
https://github.com/difi/data-docs/issues/8

jimjyang committed:
Update spesifikasjon-felter.adoc

https://github.com/difi/data-docs/issues/8

stigbd commented on begrep-skos-ap-no/spesifikasjon-felter.adoc:
```diff
@@ -116,14 +116,14 @@ Merknad:: (ingen)
 URI til definisjon:: skosno:datastrukturterm
-SKOS (RDF turtle)-representasjon:: <> skosno:datastrutkurterm
-Datatype (med ev. URI til definisjon):: rdf:Literal
+SKOS (RDF turtle)-representasjon:: <> skosno:Datastrutkurterm
```
skosno:Datastrutkurterm skal vel være skosno:Datastrukturterm?

jimjyang replied:
Ja, selvsagt!

jimjyang commented on begrep-skos-ap-no/spesifikasjon-felter.adoc:
```diff
@@ -116,14 +116,14 @@ Merknad:: (ingen)
 URI til definisjon:: skosno:datastrukturterm
-SKOS (RDF turtle)-representasjon:: <> skosno:datastrutkurterm
-Datatype (med ev. URI til definisjon):: rdf:Literal
+SKOS (RDF turtle)-representasjon:: <> skosno:Datastrutkurterm
```
SKOS (RDF turtle)-representasjon:: <> skosno:Datastrukturterm

jimjyang replied:
SKOS (RDF turtle)-representasjon:: <> skosno:Datastrukturterm

jimjyang replied:
SKOS (RDF turtle)-representasjon:: <> skosno:Datastrukturterm

jimjyang committed:
Update spesifikasjon-felter.adoc

Som svar på Stig sin kommentar: skrivefeil "skosno:Datastrutkurterm skal vel være skosno:Datastrukturterm"
